### PR TITLE
Added warnings to packagemanager upgrades

### PR DESCRIPTION
**Pull request by EParzefall**
This fixes issue #2631 by providing the appropriate warnings to users relying on their distribution's package manager for upgrades.
Backport to 10.7 and 10.6 needed.

### CODE REVIEW

File: modules/admin_manual/pages/maintenance/package_upgrade.adoc [hunk 1]
```diff
@@ -1,21 +1,30 @@
 = Upgrade ownCloud From Packages
 :toc: right
+:packages-url: https://owncloud.com/older-versions/#server
 
-== Upgrade Quickstart
+== Upgrade Steps
 
 The alternative to a manual upgrade is configuring your system to use ownCloud’s
 https://download.owncloud.org/download/repositories/stable/owncloud/[Open Build Service] repository.
```
Comment (mmattel) [inline modules/admin_manual/pages/maintenance/package_upgrade.adoc:8]: This is just a reminder for later
`https://download.owncloud.org` cant be easily renamed to a `.com` address as it is with some sub-paths physically not present ! Needs to be evaluated to get rid of.

File: modules/admin_manual/pages/maintenance/package_upgrade.adoc [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
+
+Before installing upgraded packages, perform the following steps: 
 
 * Disable all xref:maintenance/manual_upgrade.adoc#review-third-party-apps[third-party apps].
 * Make xref:maintenance/backup.adoc[a fresh backup].
-* Upgrade your ownCloud packages.
-* Run xref:configuration/server/occ_command.adoc#command-line-upgrade[occ upgrade]
-+
+
+Now you can upgrade your ownCloud packages, then run xref:configuration/server/occ_command.adoc#command-line-upgrade[occ upgrade].
```
Comment (mmattel) [inline modules/admin_manual/pages/maintenance/package_upgrade.adoc:22]:
```suggestion
Now you can upgrade your ownCloud packages, then run xref:configuration/server/occ_command.adoc#command-line-upgrade[{occ-command-example-prefix} upgrade].
```
This should render properly, but may need a check. People who just c/p would not be satisfied.

**Review by mmattel — APPROVED** [review]
LGTM, only minor things

**Review by phil-davis — APPROVED** [review]
LGTM

File: modules/admin_manual/pages/maintenance/package_upgrade.adoc [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
+== Upgrade Steps
 
 The alternative to a manual upgrade is configuring your system to use ownCloud’s
 https://download.owncloud.org/download/repositories/stable/owncloud/[Open Build Service] repository.
-Then stay current by using your Linux package manager to install fresh ownCloud packages. 
-After installing upgraded packages you must run a few more steps to complete the upgrade. 
-These are the basic steps to upgrading ownCloud:
+Then stay current by using your Linux package manager to install fresh ownCloud packages. However, you should exclude the owncloud_files package during system upgrades. For more information, check out the section on xref:installation/linux_packetmanager_install.adoc[Linux Package Manager Installation]
+
+NOTE: This approach should not be used unattended nor in clustered setups.
```
Comment (mmattel) [inline modules/admin_manual/pages/maintenance/package_upgrade.adoc:11]:
```suggestion
IMPORTANT: This approach should not be used unattended nor in clustered setups.
```
I think that we can raise the level of visibility

File: modules/admin_manual/pages/maintenance/package_upgrade.adoc [hunk 4]
```diff
@@ -1,21 +1,30 @@
...
 
-== Upgrade Quickstart
+== Upgrade Steps
 
 The alternative to a manual upgrade is configuring your system to use ownCloud’s
 https://download.owncloud.org/download/repositories/stable/owncloud/[Open Build Service] repository.
-Then stay current by using your Linux package manager to install fresh ownCloud packages. 
-After installing upgraded packages you must run a few more steps to complete the upgrade. 
-These are the basic steps to upgrading ownCloud:
+Then stay current by using your Linux package manager to install fresh ownCloud packages. However, you should exclude the owncloud_files package during system upgrades. For more information, check out the section on xref:installation/linux_packetmanager_install.adoc[Linux Package Manager Installation]
```
Comment (EParzefall) [inline modules/admin_manual/pages/maintenance/package_upgrade.adoc:9]:
```suggestion
Then stay current by using your Linux package manager to install fresh ownCloud packages. However, you should exclude the ownCloud package during system upgrades. For more information, check out the section on xref:installation/linux_packetmanager_install.adoc[Linux Package Manager Installation]
```

File: modules/admin_manual/pages/maintenance/package_upgrade.adoc [hunk 5]
```diff
@@ -1,21 +1,30 @@
...
 * Make xref:maintenance/backup.adoc[a fresh backup].
-* Upgrade your ownCloud packages.
-* Run xref:configuration/server/occ_command.adoc#command-line-upgrade[occ upgrade]
-+
+
+Now you can upgrade your ownCloud packages, then run xref:configuration/server/occ_command.adoc#command-line-upgrade[{occ-command-example-prefix} upgrade].
+
 NOTE: The optional parameter to skip migration tests was removed in ownCloud 10.0. See xref:maintenance/upgrade.adoc[Testing a Migration] for background information.
 
+After the the upgrade is finished, perform the following actions:
```
Comment (ho4ho) [inline modules/admin_manual/pages/maintenance/package_upgrade.adoc:26]: @EParzefall @mmattel:

`the the` twice here.

Comment (mmattel) [reply modules/admin_manual/pages/maintenance/package_upgrade.adoc:26]: @hoho thanks for catching

Comment (EParzefall) [reply modules/admin_manual/pages/maintenance/package_upgrade.adoc:26]: Yes, thank you @ho4ho. Fixed in the latest PR #3563.

**Pull request merged**